core/jazz_environment-event-handler/index.js

```diff
@@ -597,6 +590,239 @@ function triggerBuildJob(payload, serviceDetails, configData) {
   });
 }
 
+function createSafe(environmentPayload, service_id, configData, authToken) {
+  return new Promise((resolve, reject) => {
+    var updatePayload = {};
```

Comment:
Should we have a check to see if T-Vault is enabled here?

Reply:
check already provided in wrapping function 'addSafe' of this.